pagopa/io-app
# feat: [IOBP-80] Add torch button to QRCode scan screen header

### Depends on https://github.com/pagopa/io-app/pull/4718

## Short description
This PR adds the support for torch in the QRCode scan screen.

https://github.com/pagopa/io-app/assets/6160324/b3da3d71-5149-4733-83ba-e70f1d330433


## List of changes proposed in this pull request
- Added translation keys for camera permission request;
- Added new APIs for torch to `IOBarcodeScanner`;

## How to test
**Important notice**: on simulators the torch is not available and the button will not be displayed.
On a physical device, navigate to the QRCode scan screen and tap on the torch icon in the upper-right corner of the screen. The torch of the phone should turn on/off on tap.
